app/src/main/java/com/sopt/now/presentation/ui/MainActivity.kt

```diff
@@ -1,23 +1,50 @@
 package com.sopt.now.presentation.ui
 
+import android.content.Intent
 import android.os.Bundle
+import androidx.activity.OnBackPressedCallback
 import androidx.appcompat.app.AppCompatActivity
 import com.sopt.now.R
 import com.sopt.now.data.User
 import com.sopt.now.databinding.ActivityMainBinding
 import com.sopt.now.presentation.utils.KeyStorage
 import com.sopt.now.presentation.utils.getSafeParcelable
+import com.sopt.now.presentation.utils.showToast
 
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
     private var user: User? = null
+    private var backPressedTime: Long = 0
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
         getUserInfo()
         showUserInfo()
+
+        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+                if (System.currentTimeMillis() - backPressedTime < 2000) {
```

Comment:
```suggestion
                if (SystemClock.elapsedRealtime() - backPressedTime < 2000) {
```

안정성 말고 별 차이는 존재하지 않지만 간격 측정이나 타이머 같은 경우 이렇게 쓰기도 한답니다~